bpf/process/bpf_process_event.h

```diff
@@ -457,7 +457,8 @@ get_caps(struct msg_capabilities *msg, struct task_struct *task)
 {
 	const struct cred *cred;
 
-	probe_read(&cred, sizeof(cred), _(&task->real_cred));
```

Comment:
can you add some notes to the commit message on what cred and real_cred are and why one is preferred to the other.